gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -698,6 +764,10 @@ gst_tensor_filter_install_properties (GObjectClass * gobject_class)
           "Set channel first (NCHW) or channel last layout (NHWC) or None for output data. "
           "Layout of the data can be any or NHWC or NCHW or none for now. ",
           "", G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
+  g_object_class_install_property (gobject_class, PROP_OUTPUTRANKS,
+      g_param_spec_string ("outputranks", "Rank of Out Tensor",
+          "The Rank of the Out Tensor, which is separated with ',' in case of multiple Tensors",
+          "", G_PARAM_READWRITE | G_PARAM_STATIC_STRINGS));
```

Comment:
need to remove additional properties

Reply:
My fault.
Instead of removing this property, I set it as READ_ONLY permission.
* G_PARAM_READWRITE -> G_PARAM_READABLE